src/test/java/org/gridsuite/shortcircuit/server/dto/ShortCircuitParametersInfosTest.java

```diff
@@ -0,0 +1,75 @@
+package org.gridsuite.shortcircuit.server.dto;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.powsybl.shortcircuit.ShortCircuitParameters;
+import com.powsybl.shortcircuit.VoltageRange;
+import lombok.NonNull;
+import lombok.SneakyThrows;
+import org.assertj.core.api.WithAssertions;
+import org.gridsuite.shortcircuit.server.RestTemplateConfig;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.gridsuite.shortcircuit.server.service.ShortCircuitService.CEI909_VOLTAGE_PROFILE;
+
+@ContextConfiguration(classes = { RestTemplateConfig.class })
+@JsonTest
+class ShortCircuitParametersInfosTest implements WithAssertions {
```

Comment:
What does it bring to implement WithAssertions ?

Reply:
<https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/WithAssertions.html>

It's a way to not have to manage the `import static`s.